dm_cli/import_package.py

```diff
@@ -178,6 +182,8 @@ def upload_global_file(address: str) -> str:
                         document = resolve_local_ids_in_document(document)
                         print(f"Successfully resolved local IDs in:\t{data_source}{name}")
                     dmss_api.document_add_simple(data_source, document)
+                except ApplicationException as error:
+                    raise error
```

Comment:
Why are we catching to just raise it again?

Reply:
Yes, so that it does not fall into the one bellow :)